agent/pkg/proto/agent.proto

```diff
@@ -0,0 +1,272 @@
+syntax = "proto3";
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/pulumi/pulumi-kubernetes-operator/agent/pkg/proto";
+
+package agent;
```

Comment:
I think this should be `package proto` to match its subdirectory but I could be wrong. It might be more expected to have this live in the top-level agent dir with generated code down here.